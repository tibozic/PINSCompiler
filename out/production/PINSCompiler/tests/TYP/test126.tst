fun f(x: integer, y:integer, z: string): string = x {
where fun g(x: integer, y: integer, z:string): string = z
}
