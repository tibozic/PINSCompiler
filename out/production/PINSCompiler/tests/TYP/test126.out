[1:1-3:2]: ERROR: Function return type doesn't match function body return type: `str`!=`int`

